static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-epic-aud-support.js

```diff
@@ -0,0 +1,71 @@
+// @flow
+import { makeABTest } from 'common/modules/commercial/contributions-utilities';
+
+import {
+    getSupporterPaymentRegion as geolocationGetSupporterPaymentRegion,
+    getSync as geolocationGetSync,
+} from 'lib/geolocation';
+
+import config from 'lib/config';
+
+import type { CtaUrls } from 'common/modules/commercial/contributions-utilities';
+
+const AUDsupportURL = 'https://support.theguardian.com/au';
+const abTestName = 'AcquisitionsEpicAudSupport';
+
+const oneButtonTemplate = (urls: CtaUrls): string => {
+    const url = urls.supportUrl || '';
+
+    const supportButtonSupport = `
+        <div>
+            <a class="contributions__option-button contributions__contribute contributions__contribute--epic contributions__contribute--epic-member contributions__contribute--epic-single-button"
+              href="${url}"
+              target="_blank">
+              Support the Guardian
+            </a>
+        </div>`;
+
+    const paymentLogos = `<img class="contributions__payment-logos contributions__contribute--epic-member" src="${config.get(
+        'images.acquisitions.paypal-and-credit-card',
+        ''
+    )}" alt="Paypal and credit card">`;
```

Comment:
We have this bit of code in 3 places now so it could be worth putting it into it's own template

Reply:
Yes, I totally agree. I am planning to do a PR adding types to the options object for AB tests, I can include this small refactor there.